presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -110,13 +121,20 @@ public Module toModule()
     {
         private static final String CONTAINER_REPORTS_DIR = "/docker/test-reports";
 
+        // CI steps fail after 2h, starting environment should not take more than 30m
+        private static final long TEST_RUN_STARTUP_TIMEOUT = Duration.valueOf("30m").toMillis();
+        private static final long BACKOFF_MIN = 100L;
+        private static final long BACKOFF_MAX = Duration.valueOf("5m").toMillis();
```

Comment:
5m is very generous. What backoff is supposed to help with?
Supposedly we're the only player in the system.

Reply:
This backoff is only to put some delay between retries. As delay doubles on every retry it won't reach 5 minutes in default configuration.